### PR TITLE
Fix parameter combination print for SGVectors

### PR DESCRIPTION
**Pull request by sorig**
See issue #2819

### CODE REVIEW

File: src/shogun/modelselection/ParameterCombination.cpp [hunk 1]
```diff
@@ -316,22 +318,21 @@ void CParameterCombination::print_tree(int prefix_num) const
 				SG_SPRINT("\"%s\":%s at %p ", param->m_name,
 						current_sgobject->get_name(), current_sgobject);
 			}
-
-		    else if (m_param->get_parameter(i)->m_datatype.m_ctype == CT_SGVECTOR)
+		    else if (ctype == CT_SGVECTOR || ctype == CT_VECTOR)
```
Comment (lisitsyn) [inline src/shogun/modelselection/ParameterCombination.cpp]: Does it correspond to the if at line 314? Something strange with indentation

**Comment by sorig** [comment]
@lisitsyn The indentation issue is fixed now (the indentation in the original version of the file is also weird). The fix is simply to add an extra check for the container type being `CT_VECTOR` in the the if statement at [line 320 in the original version of the file](https://github.com/shogun-toolbox/shogun/blob/develop/src/shogun/modelselection/ParameterCombination.cpp#L320)

**Comment by karlnapf** [comment]
Seems fine to me!

**Comment by karlnapf** [comment]
And travis errors are unrelated too, merging
Thanks!

**Pull request merged**